.github/workflows/linux.yml

```diff
@@ -0,0 +1,122 @@
+name: Build Linux
+
+on:
+  # We run this once a week so we regularly publish pyvalhalla-weekly
+  # but not on _every_ master commit
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
it would probably be easier to move python related stuff into a separate GitHub Action